client/client.go

```diff
@@ -166,3 +262,15 @@ func (o *OpkClient) OidcAuth(
 	}
 	return pkt, nil
 }
+
+func (o *OpkClient) GetSigner() crypto.Signer {
+	return o.signer
+}
+
+func (o *OpkClient) GetAlg() jwa.KeyAlgorithm {
+	return o.alg
+}
+
+func (o *OpkClient) GetSignGQ() bool {
+	return o.signGQ
+}
```

Comment:
What's the value of returning this variable?

Reply:
Good question:

1. It is an important property of the client so it seems likely that a developer may wish to check it is set the way they expect it.

2. client_test.go needs it to test that the option is set correctly https://github.com/openpubkey/openpubkey/pull/93/files#diff-43e8423bcc9bb4d9fcd3ce8da7e0197e64d49376e957292eadba0316d57b000dR73